IMU.cpp

```diff
@@ -0,0 +1,62 @@
+#include "IMU.h"
+
+IMU::IMU()
+{
+
+  // Call imu.begin() to verify communication and initialize
+  if (imu.begin() != INV_SUCCESS)
+  {
+    while (1){
+      delay(5000);
+    }
+  }
+
+  imu.setSensors(INV_XYZ_ACCEL); //Enable Gyro and Accel
+  imu.setAccelFSR(2);
+    
+  imu.dmpBegin(DMP_FEATURE_6X_LP_QUAT | 
+               DMP_FEATURE_SEND_RAW_ACCEL
+               , 10);
+}
+
+//Calculates the angle from the current gravity vector away from x
+//Outputs radians or degrees depending 
+float IMU::calcAngleFromX(bool degrees)
+{
```

Comment:
I recommend following a consistent code style with the rest of the project. So 4 space tabs, etc. @xinkecf35 would know what style you are following.

Reply:
The reason I have the lines like that for dmpBegin is that there used to be much more parts activated and this made it easier to tell what was on and what was off.